manifests/install.pp

```diff
@@ -6,28 +6,25 @@
       'cyrus-sasl-plain',
       'libvirt-devel',
       'sqlite-devel',
-      "${katello_devel::scl_postgresql}-postgresql-devel",
+      katello_devel::package('postgresql-devel', $katello_devel::scl_postgresql),
       'libxslt-devel',
       'systemd-devel',
       'libxml2-devel',
       'git',
-      "${katello_devel::scl_nodejs}-npm",
+      katello_devel::package('npm', $katello_devel::scl_nodejs),
       'libcurl-devel',
       'gcc-c++',
       'libstdc++',
-      "${katello_devel::scl_postgresql}-postgresql-debversion",
-      "${katello_devel::scl_postgresql}-postgresql-evr",
+      katello_devel::package('postgresql-debversion', $katello_devel::scl_postgresql),
+      katello_devel::package('postgresql-evr', $katello_devel::scl_postgresql),
       'katello-selinux',
+      'make',
+      katello_devel::package('ruby-devel', $katello_devel::scl_ruby),
+      katello_devel::package('rubygem-bundler', $katello_devel::scl_ruby),
```

Comment:
If you do this, you perhaps the RVM support should be dropped? I wouldn't mid that at all btw but I don't know if anyone is still using it.

Reply:
I have been considering that, just didn't want to muddy the waters of this PR with that change. It benefits devel envs to mirror production even at how Ruby is provided level.